src/js/ControlBarItem.js

```diff
@@ -4,10 +4,14 @@ import Image from './Templates/Shared/Image'
 
 import {ReactComponent as IconSeekLeft} from '../img/icons/icon-seek-left.svg';
 import {ReactComponent as IconSeekRight} from '../img/icons/icon-seek-right.svg';
```

Comment:
```suggestion
import {ReactComponent as IconSeekLeft} from '../img/icons/icon-seek-left.svg';
import {ReactComponent as IconSeekRight} from '../img/icons/icon-seek-right.svg';
```

these icons are no longer needed in the project, correct?

Reply:
Correct. Removed in https://github.com/smartdevicelink/generic_hmi/pull/337/commits/cb0a87e8db9806fc9543502e572d3037e3669664